aw_watcher_window/macos.swift

```diff
@@ -225,13 +298,26 @@ class MainThing {
         data.url = activeTab.URL
         if let title = activeTab.title { data.title = title }
       }
+    } else if frontmost.localizedName == "Safari" {
+      debug("Safari browser detected, extracting URL and title")
+
+      let safariObject: SafariApplication = SBApplication.init(bundleIdentifier: bundleIdentifier)!
+
+      let frontWindow = safariObject.windows!()[0]
+      let activeTab = frontWindow.currentTab!
+
+      // Safari doesn't have an incognito mode, we cannot hide the url
```

Comment:
This isn't true, we just need to figure out how to tell if the current window is in private browsing mode

Reply:
For example, using menus: https://alexwlchan.net/2021/04/detect-private-browsing/

Reply:
Sorry, comment was bad. Here's what I meant to say, which I _think_ is true:

```suggestion
      // Safari doesn't allow incognito mode to be inspected, so we do not know if we should hide the url
```

Reply:
good call, added an issue for this here https://github.com/ActivityWatch/aw-watcher-window/issues/71